mesh/meshdb.go

```diff
@@ -258,7 +222,12 @@ func (m *MeshDB) writeBlock(bl *types.Block) error {
 		return fmt.Errorf("could not write transactions of block %v database %v", bl.ID(), err)
 	}
 
-	minblock := &types.MiniBlock{BlockHeader: bl.BlockHeader, TxIds: txids, ATXs: bl.ATXs}
+	atxids := make([]types.AtxId, 0, len(bl.Txs))
```

Comment:
In which flow do you end up with a block that holds a full representation of its ATXs, and need to be written to the db?

Reply:
there is  a method in sync.go named fetchFullBlocks that contains that flow